app/assets/javascripts/fae/form/_form.js

```diff
@@ -23,19 +26,17 @@ Fae.form = {
       $('.input.file').fileinputer();
     }
 
-    // Mutate DOM to support two column labels for all standard inputs
-    this.makeTwoColumnLabels();
-
     // make all the hint areas
     $('.hint').hinter();
   },
 
   makeTwoColumnLabels: function() {
     $('.input label').each(function() {
       var $element = $(this);
+      var has_no_helper_text = false;
 
       // Bail if we cannot find any helper_text
-      if (!$element.find('.helper_text').length) { return; }
+      if (!$element.find('.helper_text').length) { has_no_helper_text = true; }
```

Comment:
@aaronransley I feel like this implementation detaches the logic a bit and extends an already long function. What do you think about calling another function here?

```
if (!$element.find('.helper_text').length) { addNoHelperClass($element); }
```